### PR TITLE
Updating the ecosystem page

### LINKED ISSUE #10: Update the ecosystem page
Switch to the new card layout.

### PR DESCRIPTION
**Pull request by oraNod**
Resolves #10 

This PR makes a number of changes to sass styles and adds a reusable grid layout. This PR also moves the ecosystem details out of cards into a grid.

### CODE REVIEW

**Review by samccann — COMMENTED** [review]
I'm not sure where the hotlinks are defined, but I find them hard to find on this page. 

image

File: templates/_footer.html [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-    <span class="redhat">
-      A Red Hat sponsored project
-    </span>
-    <span class="redhat-logo">
-      <a href="https://www.redhat.com/" target="_blank"><img src="static/images/redhat_reversed.svg"></a>
-    </span>
-  </div>
+  <span class="licence">
+    <a href="https://www.redhat.com/en/about/privacy-policy">Red Hat privacy policy</a> |
+    <a href="https://www.redhat.com/en/about/all-policies-guidelines" target="_blank">All policies and guidelines</a> |
```
[the remark this answers is not in the document]
Comment (oraNod) [reply templates/_footer.html:4]: Yeah, we can figure this out in a separate PR for the footer. I need to know what goes in there. I think we need a license and copyright.

[the remark this answers is not in the document]
Comment (oraNod) [reply templates/_footer.html:4]: Here's the issue for the footer: https://github.com/ansible/jinja-docsite/issues/12

I'll get to it next week.

**Comment by oraNod** [comment]
@Andersson007 Can you please take a look and let me know how you would add collections to the ecosystem? Do you think it should link to the collection index in package docs?

Thanks.

**Comment by samccann** [comment]
Definitely like the grid. Merge when ready

**Comment by oraNod** [comment]
@Andersson007 I've added your comment to a second PR: https://github.com/ansible/jinja-docsite/issues/25

That fits outside the format of the grid a little so I'd like to do it separately. But I totally agree that collections need to be represented on the ecosystem page. They should be prominent too.

**Pull request merged**